.drone.star

```diff
@@ -129,7 +129,7 @@ def main(ctx):
     notify(ctx),
   ]
 
-  if ctx.build.cron != "":
+  if ctx.build.cron:
```

Comment:
What happens in drone - I thought that this was a string?
Can you write some comments in the PR and switch it from draft to "real".